bin/fixutf8/dune.in

```diff
@@ -0,0 +1,6 @@
+(executable
+  (name gwfixutf8)
```

Comment:
Could we only have a `public_name` ?

Reply:
I believe GeneWeb will refuse to create a person with a missing first_name!